app/src/main/java/com/hedvig/app/util/extensions/ContextExtensions.kt

```diff
@@ -78,9 +79,8 @@ fun Context.hideKeyboard(view: View) {
     inputMethodManager.hideSoftInputFromWindow(view.windowToken, 0)
 }
 
-@RequiresApi(Build.VERSION_CODES.R)
-suspend fun Context.hideKeyboardIfVisible(view: View, inputView: View, delayMillis: Long = 0) {
-    if (view.rootWindowInsets?.isVisible(WindowInsets.Type.ime()) == true) {
+suspend fun Context.hideKeyboardWithDelay(inputView: View, delayMillis: Long = 0) {
+    if (inputView.rootView.rootWindowInsets?.isVisible(WindowInsetsCompat.Type.ime()) == true) {
```

Comment:
This needs to use the compat-equivalents, or it will break 😞

Reply:
i.e the `rootWindowInsets`-compat ones

Reply:
Ah good catch, fixed